ext/nnstreamer/tensor_filter/meson.build

```diff
@@ -526,11 +526,11 @@ if get_option('enable-mediapipe')
     error('Not Supported System & Architecture. Linux x86_64 is required')
   endif
 
-  cmd = run_command('sh', '-c', 'echo $MEDIAPIPE_HOME')
+  cmd = run_command('sh', '-c', 'echo $MEDIAPIPE_HOME', check : true)
   MEDIAPIPE_HOME = cmd.stdout().strip()
 
-  mediapipe_env_exist = run_command('[', '-z', MEDIAPIPE_HOME, ']').returncode()
-  mediapipe_dir_exist = run_command('[', '-d', MEDIAPIPE_HOME, ']').returncode()
+  mediapipe_env_exist = run_command('[', '-z', MEDIAPIPE_HOME, ']', check : true).returncode()
```

Comment:
In the context of what you're trying to do here...

You do two run_command invocations, and if the following two conditions are both true:
- the first succeeds
- the second fails

you emit a manual `error()`.

In this case since you're explicitly checking the return value, you should probably use `check: false`

But, there's a bigger issue to consider. The first one can be rewritten to not use run_command at all.

```meson
mediapipe_env_exist = MEDIAPIPE_HOME != ''

# This requires bumping to Meson 0.53.0
fs_mod = import('fs')
mediapipe_dir_exist = fs_mod.is_dir(MEDIAPIPE_HOME)
```

Reply:
@gichan-jang The first line has been replaced and the other one seems to still require it so that ```check: false``` should be provided.

@eli-schwartz Thanks. I've revised my PR based on your comment.